app/femr/business/services/core/IEncounterService.java

```diff
@@ -116,7 +116,7 @@
      * @param userId id of the physician that screened the patient for diabetes, not null
      * @return updated patient encounter item and/or errors if they exist, or null if errors
      */
-    ServiceResponse<PatientEncounterItem> screenPatientForDiabetes(int encounterId, int userId);
+    ServiceResponse<PatientEncounterItem> screenPatientForDiabetes(int encounterId, int userId, Boolean isScreened);
```

Comment:
Above this method are comments that describe each parameter - we should do the same for "isScreened" as this isn't very intuitive for a future developer. These comments show up whenever you Ctrl-q this method inside IntelliJ.